README.md

```diff
@@ -1,23 +1,36 @@
 # Enarx
 
-## Introduction
-Enarx is an application deployment system enabling applications to run within Trusted Execution Environments (TEEs) without rewriting for particular platforms or SDKs. It handles attestation and delivery into a run-time “Keep” based on WebAssembly, offering developers a wide range of language choices for implementation. Enarx is CPU-architecture independent, enabling the same application code to be deployed across multiple targets, abstracting issues such as cross-compilation and differing attestation mechanisms between hardware vendors. Work is currently underway on AMD SEV and Intel SGX.
-
-We've known for a long time that we need encryption for data at rest and in transit: Enarx helps you do encryption for data as well as algorithms in use.
+## Introduction Enarx is an application deployment system enabling applications
+to run within Trusted Execution Environments (TEEs) without rewriting for
+particular platforms or SDKs. It handles attestation and delivery into a
```

Comment:
I think something along these lines would add more precision to this paragraph:

"It handles attestation and delivery of WebAssembly workloads into a run-time "Keep", [..]"

Reply:
I think that's already too specific, we want to keep this both factually correct and readable by various audiences, including some for which "WebAssembly workload" will be less clear than "based on WebAssembly", which is arguably more hand-wavy, but on purpose.

My feeling:
"WebAssembly workloads" --> Wait, do I use that? Am I doing that? Will I need to change things?
"based on WebAssembly" --> Right, I've heard of that technology, let's see how they use it.